lib/heuristics/scheduling_heuristic.rb

```diff
@@ -303,13 +304,42 @@ def update_route(route_data, first_index, first_start = nil)
       route
     end
 
+    def exist_possible_first_route_according_to_same_point_day(service_id, point_id)
```

Comment:
```suggestion
    def exist_possible_first_route_according_to_same_point_day?(service_id, point_id)
```
Whenever the returned value is a boolean I would suggest to add a question mark to the method name